### ISSUE TITLE
types like `Name` are too restrictive

### ISSUE THREAD

**Issue by shakyShane** [JoviDeCroock/hooked-head#12]
Since you'll never be able to accurately `meta name=...` to every possible name, it would be better to use these types as assistance only, rather than restrictions.

eg: Im working on a site now which uses both of these...

```
<meta name="p:domain_verify" content="..."/>
<meta name="google-site-verification" content="..." />
```

But Typescript is failing to compile because of the `Name` type

**Comment by JoviDeCroock** [comment]
Hmm, so `property|name: string | Property` so it's `string` or ones that it can autocomplete?

**Comment by shakyShane** [comment]
Well it depends on what yuor goals are, I guess.

seems impossible to enforce type-safety on a name where literally any value is possible - so yeah a union of known keys + `string` would be better for this one _imo_

**Comment by JoviDeCroock** [comment]
Sadly this does remove the auto completion https://github.com/JoviDeCroock/hooked-head/commit/08aa17dc75bdf9095a152310fc147ef80a46aa0b#diff-605ccb99878350a60eafb8e1c1fc52bfR7

**Comment by JoviDeCroock** [comment]
This has been released in 0.4.1
